lib/routes/hellogithub/index.ts

```diff
@@ -62,66 +57,21 @@ async function handler(ctx) {
         const $ = load(tagResponse.data);
 
         tag = $('meta[property="og:title"]')?.attr('content')?.split(' ').pop();
-        buildId = tagResponse.data.match(/"buildId":"(.*?)",/)[1];
-    }
-
-    if (!buildId) {
-        const buildResponse = await got({
-            method: 'get',
-            url: rootUrl,
-        });
-
-        buildId = buildResponse.data.match(/"buildId":"(.*?)",/)[1];
     }
 
-    let items = response.data.data.slice(0, limit).map((item) => ({
+    const items = response.data.data.slice(0, limit).map((item) => ({
         guid: item.item_id,
-        title: item.title,
+        title: `${item.name}: ${item.title}`,
         author: item.author,
         link: `${rootUrl}/repository/${item.item_id}`,
-        description: item.description,
         pubDate: parseDate(item.updated_at),
+        name: `${item.author}/${item.name}`,
+        summary: item.summary,
+        language: item.primary_lang,
     }));
 
-    items = await Promise.all(
```

Comment:
This essentially remove all the `description` of `/hellogithub/home`
https://github.com/DIYgod/RSSHub/pull/16823#issuecomment-2361584669

Reply:
I think /home is a section introduction, not a traversal item /detail

Reply:
This route does provide these [since the very beginning](https://github.com/DIYgod/RSSHub/blob/e4c6dabf7061fa99ce7adab0847314460bca1a1d/lib/v2/hellogithub/article.js).

Reply:
But the ones you mentioned haven't been updated for a long time, right?

Reply:
The one I mentioned do receive reasonable updates. This route is originally purposed in #8879, reworked in #11000 and received fixes in #15388. The last fix was in Apr this year. All of these can be seen from git commit history, and none of these changes tried to remove the `desription`.

Reply:
The point is that these commits did not solve the problem, and the content remained in June. This is the main purpose of my fix